README.md

```diff
@@ -1,41 +1,57 @@
-# geo-arrow-spec
+# GeoArrow - geospatial data with Apache Arrow
 
 Specifications for storing geospatial data in Apache Arrow and Apache Parquet.
 
 The [Apache Arrow](https://arrow.apache.org/) project specifies a standardized
-language-independent columnar memory format. It enables shared computational
-libraries, zero-copy shared memory and streaming messaging, interprocess
-communication, etc and is supported by many programming languages. The Feather
-file format is an on-disk representation of this memory format.
-
-[Apache Parquet](https://parquet.apache.org/) is an efficient, columnar storage
-format (originating from the Hadoop ecosystem). It is a widely used file format
-for tabular data, and its implementation for C-based languages (Python, R) is
-included in the Apache Arrow project.
-
-Geospatial data often comes in tabular format, with one (or multiple) column
-with feature geometries and additional columns with feature attributes. 
-
-Defining a standard way to store geospatial data in the Arrow memory layout
-can help interoperability between different tools and enable fast data
-transfer:
-
-- Ensure that different tools can read Parquet or Feather files with geospatial
-  data and correctly interpret the geometry information. Concrete example: we
-  want that the R `sf` package can read a parquet file produced by the Python
-  `geopandas` package and vice versa.
-- A standard in-memory specification for geometry data using the Arrow memory
-  layout. This ensures geospatial data can be used in a growing number of
-  applications that use the Arrow standard for in-memory data sharing or fast
-  data transfer.
+language-independent columnar memory format. It enables shared computational libraries,
+zero-copy shared memory and streaming messaging, interprocess communication, etc and is
+supported by many programming languages and data libraries.
+
+Spatial information can be represented as a collection of discrete objects using points,
+lines and polygons, i.e. vector data. The
+[Simple Feature Access](https://www.ogc.org/standards/sfa) standard provides a widely
+used abstraction, defining a set of geometries: Point, LineString, Polygon, MultiPoint,
+MultiLineString, MultiPolygon, GeometryCollection. Next to a geometry, simple features
+can also have non-spatial attributes that describe the feature.
+
+Geospatial data often comes in tabular format, with one (or multiple) column with
+feature geometries and additional columns with feature attributes. The Arrow columnar
+memory model is therefore perfectly suited to store such data, both vector features and
+its attribute data. The GeoArrow specification is about how the vector features
+(geometries) can be stored in Arrow (and Arrow-compatible) data structures.
+
+This repository contains the specifications for:
+
+- The memory layout for storing geometries natively in an Arrow field ([format.md](./format.md))
```

Comment:
It might be nice to have a section that explicitly describes what we mean by "native". I understand native to mean O(1) coordinate access, so excluding WKB/WKT arrays.

Reply:
I think the format.md document now has explicit sections on native vs serialized (now Dewey's PR is merged). So maybe we should clarify it there if that's not yet the case?

(removed the "native" here, since the format.md also describes both)